test/necb/unit_tests/tests/test_necb_pumppower_rules.rb

```diff
@@ -96,16 +96,16 @@ def test_NECB2015_pumppower
               max_powertoload = 22
             when 'OS_Pump_VariableSpeed'
               pumps << supplycomp.to_PumpVariableSpeed.get
-              total_pump_power += model.getAutosizedValue(supplycomp, 'Design Power Consumption', 'W').to_f
+              total_pump_power += supplycomp.getAutosizedValue('Design Power Consumption', 'W').to_f
```

Comment:
why not `supplycomp.ratedPowerConsumption`?

Reply:
good call, changed